app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayController.kt

```diff
@@ -94,7 +102,8 @@ class DefaultTabsTrayController(
     private val tabsUseCases: TabsUseCases,
     private val selectTabPosition: (Int, Boolean) -> Unit,
     private val dismissTray: () -> Unit,
-    private val showUndoSnackbarForTab: (Boolean) -> Unit
+    private val showUndoSnackbarForTab: (Boolean) -> Unit,
+    private val showCancelledDownloadWarning: (arg: CancelDownloadFragmentArguments) -> Unit,
```

Comment:
I don't think we necessarily need this `showCancelledDownloadWarning` callback function since we have the `navController` available as a property that we can just use directly in the controlleer.